src/PresentationalComponents/Dashboard/Dashboard.js

```diff
@@ -1,31 +1,110 @@
 /* eslint-disable no-console */
 import './_dashboard.scss';
-
-import { Gallery, GalleryItem } from '@patternfly/react-core/dist/js/layouts/Gallery/index';
-import { PageHeader, PageHeaderTitle } from '@red-hat-insights/insights-frontend-components/components/PageHeader';
-
+import { Card, CardHeader, CardBody, Divider, PageSection, Title } from '@patternfly/react-core';
```

Comment:
can yah break up these imports [kinda as seen here](https://github.com/RedHatInsights/insights-advisor-frontend/blob/d5be602bd2b7aca848507c9b0a3a735c6947e458/src/PresentationalComponents/RuleDetails/RuleDetails.js#L12) i'll make for a smaller bundle in the future

Reply:
still this one @christiemolloy  🍍

Reply:
Woops! Just updated